.github/workflows/modern-distributions.yml

```diff
@@ -11,19 +11,10 @@ jobs:
     permissions:
         pull-requests: write
```

Comment:
I don't think we want write access here.

Reply:
The pull_request trigger requires review before running the pipeline so I think it's acceptable in this workflow